src/openzaak/config/models.py

```diff
@@ -60,3 +61,46 @@ class Meta:
 
     def __str__(self):
         return force_str(self._meta.verbose_name)
+
+
+class CloudEventConfig(SingletonModel):
+    """
+    Configure values required for sending cloud events
+    """
+
+    enabled = models.BooleanField(
+        help_text=_("Whether or not cloudevents should be sent on Zaak create"),
+        default=False,
+    )
+    logius_service = models.ForeignKey(
+        Service,
+        null=True,
+        on_delete=models.CASCADE,
+        help_text=_("The service to which cloudevents will be sent on Zaak create"),
+    )
```

Comment:
Could you add an extra field `webhook_path` here to configure the path to which we will send cloud events? Currently this is hardcoded as `/event/zaak`, but I'm not sure what the actual path is, so let's make it configurable